bigbluebutton-client/src/org/bigbluebutton/main/views/AudioSelectionWindow.mxml

```diff
@@ -76,6 +76,12 @@ with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
 					vboxListen.percentWidth = 50;
 				}
 				
+				var browser:String = ExternalInterface.call("determineBrowser")[0];
+				if (browser != "Firefox" && browser == "Chrome") {

```

Comment:
No, this should be doing a feature detect against a webrtc 1.0 object existing - something like `if(typeof(window.RTCPeerConnection) != "undefined") {`